x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/types.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { UseFormReturn, ControllerRenderProps, ControllerFieldState } from 'react-hook-form';
+import {
+  ServiceLocations,
+  FormMonitorType,
+} from '../../../../../common/runtime_types/monitor_management';
+
+export type StepKey = 'step1' | 'step2' | 'step3' | 'scriptEdit';
+
+export interface Step {
+  title: string;
+  children: React.ReactNode;
+}
+
+export type StepMap = Record<FormMonitorType, Step[]>;
+
+export * from '../../../../../common/runtime_types/monitor_management';
+export * from '../../../../../common/types/monitor_validation';
```

Comment:
What is the concern here? I wanted to avoid super long import paths.

Reply:
i think it could be an IDE thing. In webstorm, this leads to add odd DX, since clicking on type will lead me to this file instead of where it's actually declared.